### ISSUE TITLE
dotf is not a function

### ISSUE THREAD

**Issue by anodynos** [googledatastudio/dscc-gen#23]
```
╰─$ npm --version                                                                                                                                                                        
6.4.1
╰─$ node --version
v10.14.2
╰─$ npm init
....
package name: (deleteme) 

╰─$ npm i @google/dscc-gen
+ @google/dscc-gen@2.0.7
added 323 packages from 216 contributors and audited 740 packages in 11.153s

╰─$ npx @google/dscc-gen connector
? Project name testAnodynos
? How will users authenticate to your service? NONE       - No authentication required.
dotf is not a function
```
I can see it generates a `testAnodynos` dir and a bunch of files (incl package.json) and then it gets to the error and deletes them.

**Comment by anodynos** [comment]
In the compiled code `node_modules/@google/dscc-gen/build/connector/validation.js` changing 
      `const dotf = require("dotf");   `
to 
      `const dotf = require("dotf").default;`

fixed it:-)

**Comment by diminishedprime** [comment]
Thanks for the issue, this should be resolved with [dscc-gen 2.0.9]. I'm not sure if I actually ended up changing anything, but I was also getting the `dotf is not a function` error, but I'm not any longer.  

[dscc-gen 2.0.9]: https://www.npmjs.com/package/@google/dscc-gen/v/2.0.9
